client/src/component/InputDate/index.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import 'date-fns';
+import DateFnsUtils from '@date-io/date-fns';
+import {
+  MuiPickersUtilsProvider,
+  KeyboardDatePicker,
+} from '@material-ui/pickers';
+
+const InputDate = ({ label }) => {
+  const [selectedDate, setSelectedDate] = React.useState(new Date());
+
+  const handleDateChange = (date) => {
+    setSelectedDate(date);
```

Comment:
![image](https://user-images.githubusercontent.com/32321796/113630588-c88b2d80-9670-11eb-8f72-f897fdc1398b.png)
as you see the date here is an object date, please convert it to string with the format we use in our database

Reply:
Done